package.json

```diff
@@ -86,7 +89,8 @@
     "slate-react": "0.22.10",
     "tiny-invariant": "1.0.6",
     "use-popper": "1.1.6",
-    "warning": "4.0.3"
+    "warning": "4.0.3",
```

Comment:
In a monorepo, the root `package.json` should not have any runtime dependencies, as it simply does not make sense to. Instead runtime deps should be defined by each package.

Reply:
yeah I hadn't got to that party yet 😄